data/templates/Detector_ArcAppliance.csx

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Threading;
+
+private static string GetQuery(OperationContext<ArmResource> cxt)
+{
+    return
+    $@"
+        let startTime = datetime({cxt.StartTime});
+        let endTime = datetime({cxt.EndTime});
+        cluster('ClusterName').database('DBName').YOUR_TABLE_NAME
+        | where TIMESTAMP >= startTime and TIMESTAMP <= endTime
+        YOUR_QUERY
+    ";
+}
+
+
+[ArmResourceFilter(provider: "Microsoft.ResourceConnector", resourceTypeName: "appliances")]
+[Definition(Id = "YOUR_DETECTOR_ID", Name = "", Author = "YOUR_ALIAS", Description = "")]
+public async static Task<Response> Run(DataProviders dp, OperationContext<ArmResource> cxt, Response res)
+{
+    res.Dataset.Add(new DiagnosticData()
+    {
+        Table = await dp.Kusto.ExecuteClusterQuery(GetQuery(cxt), null, "GetQuery"), 
```

Comment:
Please change this line 
FROM ==> await dp.Kusto.ExecuteClusterQuery(GetQuery(cxt), null, "GetQuery")
To ==> await dp.Kusto.ExecuteClusterQuery(GetQuery(cxt), "KustoClusterName", "KustoDatabaseName", null, "GetQuery")

This is due to a recent change in the platform. The documentation has now been updated.

Reply:
Fixed in latest commit